src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/cache/BitbucketCache.java

```diff
@@ -0,0 +1,101 @@
+package com.cloudbees.jenkins.plugins.bitbucket.server.client.cache;
+
+import com.cloudbees.jenkins.plugins.bitbucket.api.*;
+import com.cloudbees.jenkins.plugins.bitbucket.server.client.BitbucketServerAPIClient;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Logger;
+
+public class BitbucketCache {
```

Comment:
Why is this being recreated for the server? When Cloud already one, with some minor changes you could reuse.

Reply:
This already exists: https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/be28d7167c4213fc57bb8fd6982484367dcb5885/src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/Cache.java